Automattic/sensei
# Add paragraph with description in default template

Addresses https://github.com/Automattic/sensei/pull/5179#issuecomment-1141439880

### Changes proposed in this Pull Request

* It adds a paragraph to the default course template and fills it when skipping the editor wizard. [More context here](https://github.com/Automattic/sensei/pull/5179#issuecomment-1141439880)
* It renames all `"sensei-pattern-{X}"` to `"sensei-content-{X}"`, in order to unify the naming (now we have a placeholder in the CPT template besides the patterns).

### Testing instructions


* Create a new course.
* Fill a description in the editor wizard.
* Close the editor wizard or click on "Start with default layout" in the patterns step.
* Make sure the description was added to the content.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/171956745-1c27cb06-15a5-408a-abf9-3872ba637197.mov
